backend/controllers/vendorControllers.js

```
@@ -0,0 +1,182 @@
+const { User } = require("../models/userSchemas");
+const Property = require("../models/propertySchemas");
+const path = require("path");
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)